lang/pt.json

```diff
@@ -107,6 +114,7 @@
     "cat.ssl.r": "Resultados",
     "cat.ssl.results.champion": "Campeões!",
     
+    "cat.ssl.robocup22": "1° Div B<br>1° Ball placement technical challenge<br>2° Vision Blackout<br>3° Dribbling challenge",
```

Comment:
Shouldn't the challenges be in Portuguese?

Reply:
A tradução ao pé da letra dos desafios fica um pouco estranha, preferi deixar no formato em inglês msm.